### ISSUE TITLE
autoconverted parquet file have too big row groups

### ISSUE THREAD

**Issue by severo** [huggingface/datasets-server#1957]
See https://huggingface.co/datasets/imvladikon/hebrew_speech_coursera/discussions/1#6523d448b623a04e6c2f118a

**Comment by severo** [comment]
Launched the recreation of `imvladikon/hebrew_speech_coursera`.

**Comment by severo** [comment]
-> JobManagerCrashedError 😮

**Comment by severo** [comment]
Note that the issue is that the cells are too big (in bytes) and it's not related to the row groups (I was mistaken in the title)

**Comment by severo** [comment]
> row group is 564MB for 100 rows

The issue is that we don't allow big "cells". What should we do? Improve the error message? Allow big cells? Truncate?

**Comment by severo** [comment]
so: I think we should add a query parameter, like: "full: boolean", or "truncate: boolean", to /rows, /search, /filter.

**Comment by severo** [comment]
Also reported here: https://huggingface.co/datasets/UmaDiffusion/ULTIMA/discussions/1

**Comment by severo** [comment]
Somewhat related: https://huggingface.co/datasets/mikehemberger/inat_2021_train_mini_plantae

We should truncate more aggressively, even for /first-rows

**Comment by mikehemberger** [comment]
Hi,
Thanks for bringing this up. You are probably aware of this but once I click on the „Viewer“, the data is visible there.
Best,
![IMG_5035](https://github.com/huggingface/datasets-server/assets/52635858/d70c0ec2-2603-4019-be40-1e0aaa3bc612)
![IMG_5036](https://github.com/huggingface/datasets-server/assets/52635858/c51b6ca5-0a5b-46ea-8977-5f037e06bb43)

**Comment by mikehemberger** [comment]
Here is another „raw“ image dataset that I’ve uploaded via the web-interface (assuming it was faster then pushing it from a notebook).
Hope this helps
Best,
M
https://huggingface.co/datasets/mikehemberger/medicinal-plants/discussions/2#657c317f1953a4194ad0952d

**Comment by lhoestq** [comment]
The issue for https://huggingface.co/datasets/mikehemberger/inat_2021_train_mini_plantae is about first_rows truncation, not about autoconverted parquet files no ?

**Comment by severo** [comment]
yes, I brought the discussion here, but you're right, the issue is somewhat related. Maybe we can fix both at the same time though.

**Comment by severo** [comment]
Created https://github.com/huggingface/datasets-server/issues/2215 for https://huggingface.co/datasets/mikehemberger/inat_2021_train_mini_plantae

**Comment by severo** [comment]
See here too: https://huggingface.co/datasets/ideepankarsharma2003/MidjourneV6_Image_small/discussions/1

**Comment by severo** [comment]
Another one: https://huggingface.co/datasets/Libertify/stock-sight/discussions/3

**Comment by twobob** [comment]
was there any action on this?
